web/src/main/java/org/openmrs/module/web/WebModuleUtil.java

```diff
@@ -770,7 +770,7 @@ public static void stopModule(Module mod, ServletContext servletContext, boolean
 			if (root.getElementsByTagName("dwr").getLength() > 0) {
 				
 				// get the dwr-module.xml file that we're appending our code to
-				File f = new File(realPath + "/WEB-INF/dwr-modules.xml".replace("/", File.separator));
+				File f = new File(realPath, "/WEB-INF/dwr-modules.xml");
```

Comment:
I thought you need to get rid of the forward slashes, this is where Paths comes in handy